src/shapes/Text/Text.ts

```diff
@@ -1278,13 +1278,11 @@ export class Text<
    * @return {CanvasPattern} a pattern to use as fill/stroke style
    */
   _applyPatternGradientTransformText(filler: TFiller) {
-    const pCanvas = createCanvasElement(),
-      // TODO: verify compatibility with strokeUniform
-      width = this.width + this.strokeWidth,
+    // TODO: verify compatibility with strokeUniform
+    const width = this.width + this.strokeWidth,
```

Comment:
I am sure this is incorrect but we have a different approach in the pipeline anyways